java/src/test/java/org/rocksdb/DefaultEnvTest.java

```diff
@@ -88,9 +88,14 @@ public void lowerThreadPoolCPUPriority() {
 
   @Test
   public void threadList() throws RocksDBException {
-    try (final Env defaultEnv = RocksEnv.getDefault()) {
-      final Collection<ThreadStatus> threadList = defaultEnv.getThreadList();
-      assertThat(threadList.size()).isGreaterThan(0);
+    // We need to open DB first to get at least one thread in thread list.
```

Comment:
I am just wondering about this change. These tests are passing already on Linux and macOS, so is this change Windows specific? If so, should we perhaps have two variants of this test that are differentiated by JUnit assumptions based on the OS available from the Environment.java ?

Reply:
With CMake we are running tests little different. Each test is run in separate JVM. I think native code is reusing some _thread pool_ between different databases.

Reply:
Okay then.